README.md

```diff
@@ -86,6 +83,17 @@ Contribute
   automatically closes the issue if your pull request gets merged.
 
 
+## Branches
+
+The repository has the following permanent branches:
+
+ * **master** This contains the code which has been released.
```

Comment:
What is the meaning of "released" here?

Unless we actually follow a release cycle, this seems redundant.

Reply:
It's the same as deployed. What's on this website: http://ec2-52-53-177-18.us-west-1.compute.amazonaws.com/en-us/